bin/run

```diff
@@ -25,5 +25,10 @@ if (require.main === module) {
   const program = createMainCommand()
 
   // eslint-disable-next-line promise/prefer-await-to-then
-  program.parseAsync(process.argv).catch((error_) => program.onEnd(error_))
+  program
+    .parseAsync(process.argv)
+    .then(() => {
+      program.onEnd()
```

Comment:
Note: if `onEnd()` throws, it will be called twice. Is that ok?

Reply:
Great point! I don't see how it can throw https://github.com/netlify/cli/blob/c2040bb9cd846d52e38eed44232872da7bbcdc91/src/commands/base-command.js#L298

We can put everything in `onEnd` to make it clear that it shouldn't throw, WDYT?